src/components/CouchView.svelte

```diff
@@ -2,68 +2,60 @@
   import { onMount } from "svelte";
   import { state as authState } from "../auth.js";
   import { view, all_docs } from "../couch.js";
+  import CouchViewTable, { data } from "../components/CouchViewTable.svelte";
+  import SelectedView, { result } from "../components/SelectedView.svelte";
```

Comment:
You'll have to rename these imports, but remember that you can just do `"./CouchViewTable.svelte"` instead of `"../components/CouchViewTable.svelte"`.